isPosition.js

```diff
@@ -0,0 +1 @@
+module.exports = require('ssb-poll-schema').isPosition
```

Comment:
found having these methods at root really nice for going like : 

```js
const isPoll = require('scuttle-poll/isPoll') 
```

no inject needed, only need scuttle-poll installed, no ssb-poll-schema